machAeroTutorials/overset_theory.rst

```diff
@@ -133,6 +230,16 @@ More about this and the implementation of IHC in ADflow can be found `here <http
 Flood troubleshooting
 ---------------------
 
+If the mesh is flooding (too many flooding iterations, a high number of flooded cells), we need to fist prevent this to get a valid hole cutting.
+For this, we need to check leaks in the flooding process:
+
+# Set the ADflow option: ``'nrefine`:1``. This stops the IHC algorithm after one iteartion. You will get error warnings but this is fine, we just want to get an intermediate output for debugging. The ``nrefine`` option controls how many iterations of the IHC algorithm is performed. You can also modify the ``nFloodIter`` option to control how many *flooding* iterations are performed. For example, if ADflow seg-faults in the first overset iteration because the whole mesh floods, then you can stop the flooding iterations early by setting ``nFloodIter`` to 1 or 2. A value of 1 will just determine the flood-seeds, a value of 2 will do a first pass of the flooding process.
+# Set ADflow option: ``‘usezippermesh’:False`` This skips the zipper mesh generation, which may crash if the hole cutting does not work.
+# Run the overset check file: ihc_check.py (Note from Anil: I copied Ney's notes as is, I am not aware where this file is now.)
```

Comment:
The file is in the private tutorial repo

Reply:
What do you suggest we do about this? Is it worth copying over to this repo as well? Should I say this is available in the internal tutorials or sth like that?

Reply:
I would prefer if this file is publicly available if you mention it in the tutorial.

Maybe it would make sense to drop it to the ADflow repo under examples...

Reply:
From a cursory glance, it seems to just set `surfacevariables` and `volumevariables` to `['blank']` and then call `writeVolumeSolutionFile()`. So maybe we can just describe the steps here? Alternatively we can duplicate the file here and add it to the repo.

Reply:
actually yeah, so I think we can just add the file here? Do you want to avoid code duplication? If so, I can also copy the commands as a code block here.

Reply:
It looks like the script was set up specifically for the case in the private repo. If the script can be generalized, the best approach would be as @DavidAnderegg mentioned, i.e. convert it to a general script and put it in ADflow. Do you think that's doable?

Reply:
In the interest of not delaying this PR any more, I just added the ihc_check script here and added some information about it in the tutorial docs. Let me know how it looks.

Reply:
Looks fine, I'll open an issue to move it over to adflow later.

Reply:
makes sense, thanks!